--- hub-2.3.0-pre8/script/bootstrap~	2016-10-03 18:38:25.000000000 +0000
+++ hub-2.3.0-pre8/script/bootstrap	2016-12-27 00:45:15.759293260 +0000
@@ -21,18 +21,6 @@
   STATUS=1
 fi
 
-{ ruby --version
-  if [ -n "$TRAVIS" ]; then
-    script/cached-bundle install --deployment --jobs=3 --retry=3
-  else
-    bundle install --path vendor/bundle
-  fi
-  bundle binstub cucumber ronn --path bin
-} || {
-  echo "You need Ruby 1.9 or higher and Bundler to run hub tests" >&2
-  STATUS=1
-}
-
 if [ $STATUS -eq 0 ]; then
   echo "Everything OK."
 fi
--- hub-2.3.0-pre8/Makefile~	2016-10-03 18:38:25.000000000 +0000
+++ hub-2.3.0-pre8/Makefile	2016-12-27 00:47:24.534731236 +0000
@@ -65,7 +65,7 @@
 share/man/man1/hub.1.ronn:
 	true
 
-install: bin/hub man-pages
+install: bin/hub
 	bash < script/install.sh
 
 clean:
--- hub-2.3.0-pre8/script/install.sh~	2016-10-03 18:38:25.000000000 +0000
+++ hub-2.3.0-pre8/script/install.sh	2016-12-27 00:49:01.352555873 +0000
@@ -14,11 +14,11 @@
 fi
 
 prefix="${PREFIX:-$prefix}"
-prefix="${prefix:-/usr/local}"
+prefix="${prefix:-/usr/}"
 
 for src in bin/hub share/man/*/*.{1,txt}; do
   dest="${prefix}/${src}"
-  mkdir -p "${dest%/*}"
+  mkdir -p $DESTDIR/"${dest%/*}"
   [[ $src == share/* ]] && mode="644" || mode=755
-  install -m "$mode" "$src" "$dest"
+  install -m "$mode" "$src" "$DESTDIR/$dest"
 done
--- hub-2.3.0-pre8/script/install.sh~	2016-12-27 00:49:01.000000000 +0000
+++ hub-2.3.0-pre8/script/install.sh	2016-12-27 00:49:49.840966958 +0000
@@ -16,7 +16,7 @@
 prefix="${PREFIX:-$prefix}"
 prefix="${prefix:-/usr/}"
 
-for src in bin/hub share/man/*/*.{1,txt}; do
+for src in bin/hub; do
   dest="${prefix}/${src}"
   mkdir -p $DESTDIR/"${dest%/*}"
   [[ $src == share/* ]] && mode="644" || mode=755
